graphblas/operator.py

```diff
@@ -2764,6 +2840,7 @@ def op_from_string(string):
         semiring_from_string,
         # Give binary precedence over this
         select_from_string,
+        indexunary_from_string,
```

Comment:
What would one expect to get from e.g. `op.from_string("tril")`?  Should we look for select before indexunary, b/c select is more common?

And, remind me, when we register a new UDF, do we plan to create both versions, or just the opclass that is being registered?

Reply:
Yes, we should look for select before indexunary. If it exists in select, that is most likely what the user wanted. And if you do `op.from_string("rowindex")`, you will get the indexunary version because it doesn't exist in select. So anything returning bool will default to select while everything else will be apply.

For UDFs, I haven't decided yet what to do. My initial thought is to register all places where it makes sense. So always register the indexunary, and additionally register with select if it returns a bool.